src/components/table.js

```diff
@@ -867,7 +867,7 @@ export const SimpleTable = ({
                     'aria-colindex': colindex + 1, // The first column is 1
                     className: 'table-cell',
                     style: {
-                      borderTop: `1px solid ${colors.dark(0.2)}`,
+                      borderTop: i === 0 ? 0 : `1px solid ${colors.dark(0.2)}`,
```

Comment:
Have you verified that the other tables in the codebase are not strongly affected negatively by this?

Reply:
I've looked at the other forms that use this component and don't see a difference, maybe their style is overriding this setting so has no effect. I looked at the dataset browser and preview pages. It's also used by DatasetAttributes but I didn't see a way to make that view visible